suites/pacific/rados/tier-3_rados_test_osd_rebalance.yaml

```diff
@@ -0,0 +1,153 @@
+# Suite contains tests related to osd re-balance upon OSD addition / removal
```

Comment:
Thanks for splitting the test suite.

Could we rename the file to `tier-3_rados_test-osd-reblance.yaml`? This would be compliant to the naming guidelines in place.

Reply:
done.